server.js

```diff
@@ -26,27 +52,57 @@ const Empirica = {
   },
 
   onGameStart(func) {
-    config.onGameStart = func;
+    config.onGameStart = function() {
+      const results = func.apply(null, arguments);
+      handleCallbackFuncParameters(func);
```

Comment:
Wait, I misread, I thought this was done "on assignment", it is done "on call", and after we apply, and that's too late. It might be good to just have it "on call", but before the apply, just switch these 2 lines, in all of these. 👍